index.html

```diff
@@ -0,0 +1,147 @@
+					<a class="menu-button" href="#">CC Campaing</a>
+				</div>
+    	</nav>
+    	<header>
+				<h2>"Hello! Sharing World!"</h2>
+				<h1>CREATIVE COMMONS <br>GLOBAL SUMMIT <br>2022</h1>
+				<p class="text-sign">A joyful celebration believing in the value of opennes and sharing, creating a positive change with people from over 80 countries is taking place in October, Korea</p>
+				<h3 class="date-time">2022.10.15(THU) ~ 16(FRI)</h3>
+				<p class="adress">@ National Museum of Korea, Art Center Nabi and more</p>
+			</header>
+		</div>
+		<!-- descriptive section -->
+    <section class="about">
+			<h2 class="about-head">Main Program</h2>
+			<span class="line"></span>
+			<ul class="list">
+				<li class="list-option"><a href="#"><img src="images/lecture.svg" alt="lecture-icon"></i><h3>Lecture</h3><p>Listen to the speakers from various countries about the message of sharing and opening.</p></a></li>
+				<li class="list-option"><a href="#"><img src="images/exhibition.svg" alt="exhibition-icon"><h3>CC Exhibition</h3><p>Appreciate varoius creations appylying CC lisense of artists, organized from Art Center Nabi.</p></a></li>
+				<li class="list-option"><a href="#"><img src="images/forum.svg" alt="forum-icon"><h3>Forum</h3><p>Have the time to share your thoughts and opinions with experts for each topic.</p></a></li>
+				<li class="list-option"><a href="#"><img src="images/workshop.svg" alt="workshop-icon"><h3>Workshop</h3><p>Try creating your own work using open source license rather than just watching.</p></a></li>
+				<li class="list-option"><a href="#"><img src="images/creating.svg" alt="ccignite-icon"><h3>CC Ignite</h3><p>Get opportunities to network with CC affiliates around the world, alse after the summit.</p></a></li>
+			</ul>
+			<a class="join-me-button d-nodisplay" href="#">Join CC Global Summit 2022</a>
+			<a class="join-button m-nodisplay" href="#">SEE THE WHOLE PROGRAM</a>
+  	</section>
+		<!-- speaker page -->
+		<section class="speaker-page">
+			<h2 class="speakers-title">Featured Speakers</h2>
+			<div class="line"></div>
+			<!-- speaker cards -->
+			<div id="section-speakers">
+				<div class="speaker-card">
+					<img id="picture" src="images/Picture3.png" alt="speaker-foto">
+					<div id="content">
+						<h3 id="name">Lecturer Name</h3>
+						<h4 id="titles">Lercturer`s title</h4>
+						<span class="line2"></span>
+						<p id="description">Lecturer`s description</p>
+					</div>
+				</div>
+				<div class="speaker-card">
+					<img id="picture" src="images/Picture3.png" alt="speaker-foto">
+					<div id="content">
+						<h3 id="name">Lecturer Name</h3>
+						<h4 id="titles">Lercturer`s title</h4>
+						<span class="line2"></span>
+						<p id="description">Lecturer`s description</p>
+					</div>
+				</div>
+				<div class="speaker-card m-nodisplay">
+					<img id="picture" src="images/Picture3.png" alt="speaker-foto">
+					<div id="content">
+						<h3 id="name">Lecturer Name</h3>
```

Comment:
- [ ] Let's reduce the space between text and an image, see the screenshots.

### Original
![speaker-og](https://user-images.githubusercontent.com/53124548/181644694-55f38cc4-0570-45f2-98e7-094181de3dc5.png)

### Yours
![speaker-yours](https://user-images.githubusercontent.com/53124548/181644538-de8f6d6f-b8e8-4d50-bc41-bd9eedf25fa6.png)